checkout.py

```diff
@@ -5,7 +5,10 @@
 @author: vjham
 """
 
+purchased_items = []
```

Comment:
hmmmmm maybe just `items` instead of `purchased_items` since they aren't actually purchased yet